### ISSUE TITLE
I’m getting the gambling points for Watching xQc’s livestream, but hundreds disappear out of the blue

### ISSUE THREAD

**Issue by Splatitat** [pajbot/pajbot#970]
I know it sounds strange, I was skeptical the first few times it happened, but I’ve been keeping good track of my points. 20 minutes ago I checked my points and I had about 350, then I checked right now and it’s flat 100. This happens almost every day it’s so weird,

**Comment by TroyDota** [comment]
I assume you mean y didn't spend any right?

**Comment by Splatitat** [comment]
> I assume you mean you didn't spend any right?

Exactly, I’m only using them after the stream. I had 300+ points after watching the stream for a few hours, then I checked again and I had 100. Are mods able to reset your points or something?

**Comment by alazymeme** [comment]
Only users that are level 1500 or higher can adjust points. There's no other users that can adjust them. It looks like you have 10334 points to me, where are you seeing that you only have 300+ points?
